pkg/gauges/contracts/GaugeController.vy

```diff
@@ -67,8 +68,8 @@ MULTIPLIER: constant(uint256) = 10 ** 18
 admin: public(address)  # Can and will be a smart contract
 future_admin: public(address)  # Can and will be a smart contract
 
-token: public(address)  # CRV token
-voting_escrow: public(address)  # Voting escrow
+TOKEN: immutable(address) # BPT token
```

Comment:
```suggestion
TOKEN: immutable(address) # 80-20 BAL-ETH BPT token
```

There are other BPTs referenced in the system (the BPTs that are staked), so it doesn't hurt to clarify.